packages/material-ui-lab/src/DatePicker/DatePickerKeyboard.test.tsx

```diff
@@ -0,0 +1,223 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import TextField from '@material-ui/core/TextField';
+import { fireEvent, screen, act } from 'test/utils';
+import { StaticDatePicker, DesktopDatePicker, DesktopDatePickerProps } from './DatePicker';
+import { createPickerRender, adapterToUse } from '../internal/pickers/test-utils';
+import { CalendarSkeleton } from '../CalendarSkeleton';
```

Comment:
For public components, let's us the public import

```suggestion
import CalendarSkeleton from '@material-ui/lab/CalendarSkeleton';
```

Reply:
It doesn't work for now because I didn't add public files :)